articles/user-guide/basics.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Q# Basics
+description: Basic concepts of Q#
+author: gillenhaalb
+ms.author: a-gibec@microsoft.com
+ms.date: 02/28/2020
+ms.topic: article
+uid: microsoft.quantum.guide.basics
+---
+
+# Q# Basics
+
+In this section we present a brief introduction to the basic building blocks of Q#.
+
+For a quick overview of what Q# is and where it fits in as a fundamental component of the Quantum Development Kit, you can check out [What is Q#?](xref:microsoft.quantum.overview.qsharp). 
+
+## What is a quantum program?
+
+From a technical perspective, a quantum program can be seen as a particular set of classical subroutines which, when called, perform certain operations on a quantum system.
+An important consequence of that view is that a program written in Q# does not directly model qubits themselves, but rather describes how a classical control computer interacts with those qubits.
+By design, Q# thus does not define quantum states or other properties of quantum mechanics directly.
+For instance, consider the state $\ket{+} = \left(\ket{0} + \ket{1}\right) / \sqrt{2}$ discussed in the [Quantum Computing Concepts](xref:microsoft.quantum.concepts.intro) guide.
+To prepare this state in Q#, we use the facts that the qubits are initialized in the $\ket{0}$ state, and that $\ket{+} = H\ket{0}$, where $H$ is the [Hadamard](xref:microsoft.quantum.intrinsic.h) transform:
```

Comment:
I really like this paragraph, helps understand what Q# is and what it does. As a minor comment, the link for the 𝐻 unitary operator currently goes to the API documentation for the `H` operation; it may be good to disambiguate:
```suggestion
To prepare this state in Q#, we use the facts that the qubits are initialized in the $\ket{0}$ state, and that $\ket{+} = H\ket{0}$, where $H$ is the Hadamard transform, implemented by the [`H` operation](](xref:microsoft.quantum.intrinsic.h):
```
It may also make sense for the words "Hadamard transform" to link to somewhere in the concepts guide.